apps/web/src/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useRef, useState } from "react";
+import { CheckIcon } from "@heroicons/react/24/outline";
+
+import { useHover } from "~/hooks/useHover";
+import Copy from "~/icons/copy.svg";
+import { Tooltip } from "./Tooltip";
+
+export function CopyToClipboard({
+  label = "Copy to clipboard",
+  value,
+}: {
+  label?: string;
+  value: string;
+}) {
+  const [isCopied, setCopied] = useState(false);
```

Comment:
```suggestion
  const [isCopied, setIsCopied] = useState<boolean>(false);
```

Reply:
I do not dislike that idea, but it is different from the style we've been following in the rest of the components. I think it is best not to make that change to maintain consistency with the rest of the codebase

Reply:
I’d suggest changing `setCopied` to `setIsCopied` to maintain consistency with our naming convention for state variables.